fairlearn/metrics/metrics_result.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+class MetricsResult:
+    def __init__(self):
+        self.metric = None
+        self.group_metric = {}
```

Comment:
similarly, what can we expect of group_metric?

Reply:
Added comment